### ISSUE TITLE
Settings menu not displayed

### ISSUE THREAD

**Issue by Luna664** [TheWlr9/BG3-Text-Chat#1]
![image](https://github.com/user-attachments/assets/01857237-a9ff-4c68-8310-917ff61a5ce6)

**Comment by TheWlr9** [comment]
What resolution is this on?

**Comment by Luna664** [comment]
> What resolution is this on?

2560x1440

**Comment by TheWlr9** [comment]
Have you tried changing the setting to match your game resolution located in the JSON file? Should be located at `AppData\Local\Larian Studios\Baldur's Gate 3\Script Extender\Data`

**Comment by Luna664** [comment]
Yes, I've managed to move the chat window just about where I wanted with this, but the settings menu still doesn't appear.

![image](https://github.com/user-attachments/assets/be3b81d6-24fa-4bf9-920c-980957099937)

**Comment by TheWlr9** [comment]
Strange... I can't help debug this issue because I don't own a monitor wider than 1920... There are a lot of funky things going on when the game's resolution is greater than 1920...

**Comment by luigiMinardi** [comment]
@TheWlr9 I am in a `1920x1080 @ 60 Hz in 22"` monitor, running through proton in `linux` (EndeavourOS), on the `Vulkan` version of the game.
```
Kernel: Linux 6.12.8-arch1-1
WM: i3 (X11)
Display (EK220Q): 1920x1080 @ 60 Hz in 22" [External]
Monitor (EK220Q): 1920x1080 px @ 60 Hz - 477x268 mm (21.54 i)

Vulkan: 1.3.289 - NVIDIA [565.77]
OpenGL: 4.6.0 NVIDIA 565.77
```

The setting also don't appear for me, I have a friend at a monitor with a `2560x1440` that also can't see settings, but you said that with greater res its hard to see so maybe that's the reason.

I first thought it was something to do when we open the game settings, what is this settings that you have and where is it, if you can show some screenshot, so that I can try to see if I have any insight to solve it :)

Edit: Steam Launch Options on linux `WINEDLLOVERRIDES="DWrite.dll=n,b" %command%`, no forced compatibility tool, so running with current latest stable `proton 9.0-4`.

**Comment by TheWlr9** [comment]
@luigiMinardi does it look the same as the above screenshot? The settings button should appear just on the right of the text input field, and the text input field should be flush with the left side

**Comment by luigiMinardi** [comment]
@TheWlr9 yes it looks just like the screenshot above

**Comment by TheWlr9** [comment]
Weird... Does changing the settings in the file above work for you as well? Because I'm not sure I can help debug the issue. I only have a 1080p monitor and rendering at that level. I'm unsure why you're having the issue... Do you have other mods that make use of IMGUI for their windows? And if so, do they have similar issues? Because then it might be an issue with Proton

**Comment by luigiMinardi** [comment]
@TheWlr9 
I think I can use the file to edit the position, yeah, but that is not the settings GUI, which would be better QoL. The only mods I have is the text chat, the DLL that the text chat depends on to make the Lua APIs work and a hair mod which I assume that should not affect this one? Oh I guess I did install an "Enhanced UI" mod or something like that, maybe its that one?

**Comment by luigiMinardi** [comment]
>  Do you have other mods that make use of IMGUI for their windows? 

What is IMGUI btw? I use `pyenv` through `asdf` to manage my python `$PATH`, I think my system also has a default python too which if I do its on latest python because arch is usually on most recent stable versions of everything. I do have other versions of python on `asdf`, which one of them is what is set globally to be used by basically everything.

Is there a way to check python version (and preferably `$PATH`) in game? So that I can check what Steam Proton is using.

Or is that any mod that you know that uses the IMGUI so that I can test to see if I can replicate the issue on them?

**Comment by TheWlr9** [comment]
I don't think Python is the issue here, but you can try getting the Mod Manager mod. That uses IMGUI as well

**Comment by TheWlr9** [comment]
This bug should be fixed with [This PR](https://github.com/TheWlr9/BG3-Text-Chat/pull/2)
